src/migen_axi/integration/soc_core.py

```diff
@@ -54,11 +55,16 @@ def __init__(self, platform,
             ddr=platform.request("ddr"),
         ))
 
+        # csr_addr_extension is by how many address lanes over csr
+        # should be widened to accommodate for any additional
+        # memories in axi2csr
```

Comment:
Can't it be determined automatically based on the memory ports that have been added?